subcommand/server-acl-init/command.go

```diff
@@ -291,77 +268,47 @@ service_prefix "" {
 
 	// Create catalog sync token if necessary
 	if c.flagCreateSyncToken {
-		// Create agent policy
-		syncRules := `node_prefix "" {
-   policy = "read"
-}
-node "k8s-sync" {
-	policy = "write"
-}
-service_prefix "" {
-   policy = "write"
-}`
-
-		syncPolicy := api.ACLPolicy{
-			Name:        "catalog-sync-token",
-			Description: "Catalog Sync Token Policy",
-			Rules:       syncRules,
-		}
-
-		aclSyncPolicy, _, err := consulClient.ACL().PolicyCreate(&syncPolicy, &api.WriteOptions{})
-		if err != nil {
-			c.UI.Error(fmt.Sprintf("Error creating catalog sync policy: %s", err))
+		if err := c.createACL("catalog-sync", syncRules, consulClient); err != nil {
+			c.UI.Error(err.Error())
 			return 1
 		}
+	}
 
-		sToken := api.ACLToken{
-			Description: "Catalog Sync Token",
-			Policies:    []*api.ACLTokenPolicyLink{&api.ACLTokenPolicyLink{Name: aclSyncPolicy.Name}},
-		}
-
-		// Create catalog sync token
-		syncToken, _, err := consulClient.ACL().TokenCreate(&sToken, &api.WriteOptions{})
-		if err != nil {
-			c.UI.Error(fmt.Sprintf("Error creating catalog sync token: %s", err))
+	// Create enterprise license token if necessary
```

Comment:
Don't think this comment is necessary since the below if statement says the same thing.

Reply:
Not strictly necessary, but I like using comment lines to break up chunks of code into logical blocks.